src/main/cpp/SDLPlayData.cpp

```diff
@@ -1003,7 +1009,7 @@ int main(int argc, char **argv) {
 	avformat_network_init();
 	av_log(NULL, AV_LOG_WARNING, "Init Network\n");
 
-	input_filename = "counter.mp4";
+	input_filename = "Nature_30fps_1080p.mp4";
```

Comment:
Where did you get this video file?  Please paste the URL here. Thanks.

Reply:
I sent you two links to videos that I am using for testing